Tests/Cosmos.Compiler.Tests.Bcl/Kernel.cs

```diff
@@ -44,6 +44,8 @@ protected override void Run()
                 UnsafeCodeTest.Execute();
                 DelegatesTest.Execute();
 
+                //GuidTest.Execute();
```

Comment:
Why is the test commented? When a test doesn't pass, you shouldn't comment it, but read the error logs and try to fix it.